CruzHacks/cruzhacks-2024-website
# Feature/hacker applications

# Scope

Create Hacker Application Form.

closes #18 

# Implementation

- created routes for each section
- used react hook form and zod to create section schemas
- constructed schema from section schema for each form step.

# How to Test

- Navigate to `/apply` and fill out a hacker application
- Make sure emulator or backend is running / up to date
